features/step_definitions/logging.rb

```diff
@@ -741,16 +746,18 @@
     end
   end
 
+
+  step %Q/logging collector name are stored in the collector_name clipboard/
```

Comment:
`are`--> `is`,  `collector_name` --> `:collector_name`
And same comment for all the lines calling this step.